services/motion/builtin/move_request.go

```diff
@@ -805,3 +857,26 @@ func (mr *moveRequest) stop() error {
 	}
 	return nil
 }
+
+// getExistingGeometries returns elements of the worlstate which are not transient.
+// Depending on the caller, the geometries returned are either in their relative position
+// with respect to the base or in their absolute position with respect to the world.
+func (mr *moveRequest) getExistingGeometries(transformBy spatialmath.Pose) (*referenceframe.GeometriesInFrame, error) {
```

Comment:
I think I agree with @raybjork that this function probably doesn't need to exist.

It's called twice, once with a zero pose.

The place where it's called with a zero pose can call `ObstaclesInWorldFrame` directly.

The contents of the function can be placed in-line at the second call site.